### PR TITLE
Add Disclaimer as provided by Murray

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -205,6 +205,37 @@
         example. Providing createOffer/createAnswer avoids this
         problem.</t>
       </section>
+      <section anchor="sec.disclaimer" numbered="true" toc="default">
+        <name>Contradiction regarding bundle-only "m=" sections</name>
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:209]: Should this be in caps, like the other headings? @ajeanmahoney 
```suggestion
        <name>Contradiction Regarding Bundle-only "m=" Sections</name>
```

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -205,6 +205,37 @@
...
+        and <xref target="RFC8843" format="default"/> respectively)
+        Rather than delaying publication further to come
+        to a resolution, the documents are being published as they were
+        originally approved.  The IETF intends to restart work on these
+        technologies, and revised versions of these documents will be
+        published as soon as a resolution becomes available.
+        </t>
+        <t>
+        The specific issue involves the handling of "m=" sections that
+        are designated as bundle-only, as discussed in [rfced: This RFC], Section
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:223]: should this be an xref?

Comment (ekr) [reply draft-ietf-rtcweb-jsep.xml:223]: Fine by me, but I will defer to Murray.

Comment (juberti) [reply draft-ietf-rtcweb-jsep.xml:223]: @mskucherawy

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -205,6 +205,37 @@
         example. Providing createOffer/createAnswer avoids this
         problem.</t>
       </section>
+      <section anchor="sec.disclaimer" numbered="true" toc="default">
+        <name>Contradiction Regarding Bundle-only "m=" Sections</name>
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml:209]: Based on the discussion, seems like my suggestion should be reverted.

**Review by juberti — APPROVED** [review]
Updated text with RFC Editor version, approving

**Pull request merged**